src/lib/execute-action.js

```diff
@@ -0,0 +1,2 @@
+module.exports = (action, instanceUrl, extraArgs, projectDir) => require(`../fn/${action}`)(projectDir, instanceUrl, extraArgs);
```

Comment:
Some of these parameters (instanceUrl and extraArgs in particular) are a bit nuianced and likely quite unknown to the user. Instead of exporting this internal function, what do you think of just exporting `main.js` ... it takes `(argv, env)` as its parameters so then the medic-conf library in-code would have the exact same user-interface as the medic-conf command-line tool. Then we only need to worry about one user-facing interface ...

Reply:
Yeah that's an easy enough change. If we are exporting main would it even be beneficial to make this change? I'm trying to understand what difference it would make as opposed to running a child process since it's majorly built as a CLI.

Reply:
I still think it is worthwhile. It is pretty annoying to spin up a process, monitor its lifecycle, echo its console output, etc.  Maybe I don't fully understand your endgame for this change?

Another option is to make an `index.js` and export multiple things. So for example we could export `{ run: main, execute: executeAction }` or alike. We just need to be very careful about the things we expose because changing detail of them technically becomes a breaking change for medic-conf.

Reply:
I see what you mean now. I'll rework to have main.js the main file for package.json